overrides.nix

```diff
@@ -1115,6 +1115,34 @@ self: super:
     }
   );
 
+  torch = super.torch.overrideAttrs (old:
```

Comment:
You could use `lib.makeOverridable`, that way users could provide their own override that overrides this flags in the `overrides`.

Reply:
Like this?

Reply:
Maybe? Does it work for you?

Reply:
Yes, it works for me. At least the default setting without cuda. I don't have the setup to test the CUDA support.

Reply:
I feel a bit uneasy to have untested code. Could you maybe leave a TODO which points to the commit where you introduce CUDA? Than someone who can actually test built it with CUDA can pick it up.

Reply:
Or do you mean you don't have a graphics card to test it?

Reply:
Yes, that is what I mean. It compiles fine. I just noticed an issue where the cuda/no-cuda logic was inverted though, that is fixed now. I just cannot test the cuda version at runtime.